src/Ast.ml

```diff
@@ -4,7 +4,9 @@ type idT = string [@@deriving equal, sexp]
 
 (** Types *)
 module Type = struct
-  type t =
+  type t = ast Location.location
+
+  and ast =
```

Comment:
```suggestion
  type t = t' Location.located

  and t' =
```
Please make some changes like this one below.

Reply:
Done